apps/sms/test/unit/thread_list_ui_test.js

```diff
@@ -173,16 +163,6 @@ suite('thread_list_ui', function() {
       MockOptionMenu.mTeardown();
     });
 
-    test('show settings/cancel options when list is empty', function() {
-      ThreadListUI.setEmpty(true);
-      ThreadListUI.showOptions();
-
-      var optionItems = MockOptionMenu.calls[0].items;
-      assert.equal(optionItems.length, 2);
-      assert.equal(optionItems[0].l10nId, 'settings');
-      assert.equal(optionItems[1].l10nId, 'cancel');
-    });
-
```

Comment:
Hi @julienw ,
I tried to add a test case like 'test(settings button is displayed when list is empty...' and with assert like 'assert.equal(settingsButton.style.display, 'flex'); and assert.equal(optionsButton.style.display, 'none');' .But there was an error as 'expected ' ' to be 'flex/none' ' . Is this test case needed?
Thank you.